### PR TITLE
Allow to close assistant/help and remember the option

### LINKED ISSUE #86: Close assistant/help redirects into error
Closing assistant/help on top of page redirects to `/settings/general` but this page does not exist.

### PR DESCRIPTION
**Pull request by Noles**
Add assistant vue component and allow closing.

Fixes #86.

### CODE REVIEW

File: resources/assets/js/components/Assistant.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<template>
+    <div :class="['notification', 'is-radiusless', 'animated', type ? `is-${type}` : '']" v-if="show">
```
Comment (Indemnity83) [inline resources/assets/js/components/Assistant.vue]: Can we set 'invisible' in here too, and remove it programmatically once Vue is loaded? Right now if show == false, the browser will still render this for a tick before Vue can hide it which is an annoying quirk of vue, but not something we can't overcome.

File: resources/assets/js/components/Assistant.vue [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            };
+        },
+        props: {
+            type: String,
+            title: String,
+        },
+
+        mounted() {
+            if (typeof (Storage) !== "undefined" && localStorage.getItem("assistantShow") !== null) {
+                if (localStorage.getItem("assistantShow") === "false") {
```
Comment (Indemnity83) [inline resources/assets/js/components/Assistant.vue]: Would be really nice to be able to track each notification individually. The purpose of these notifications is to guide a first time user through the system, if they close the first one of these they may not realize/see the other notifications through the system.

Maybe use a 'key' prop?

**Review by Indemnity83 — DISMISSED** [review]
This is great, even easier than what i had been planning. I made a couple comments though of things I'd like to see before merging this in.

File: resources/assets/js/components/Assistant.vue (hunk 2 above)
Comment (Noles) [reply resources/assets/js/components/Assistant.vue]: Good idea. What do you think about this? Note:Its currently only working on builds and modpacks page. For the show case.

File: resources/assets/js/components/Assistant.vue (hunk 1 above)
Comment (Noles) [reply resources/assets/js/components/Assistant.vue]: I am new to Vue but are you sure that this is happing this way? I have problem with the page loading but in google cache I found this infos: [Cache](https://webcache.googleusercontent.com/search?q=cache:O8gJtUWNpuEJ:https://forum-archive.vuejs.org/topic/12/v-if-vs-v-show/3+&cd=2&hl=de&ct=clnk&gl=at&client=ubuntu)

Comment (Indemnity83) [reply resources/assets/js/components/Assistant.vue]: v-cloak was what I was thinking of; just couldn't remember the name when I made the original comment.

**Comment by Indemnity83** [comment]
I finished up the concept locally and merged in ade8fbe69923376d202def9d009f260290eadc9c

Thanks!

File: resources/assets/js/components/Assistant.vue (hunk 1 above)
Comment (Noles) [reply resources/assets/js/components/Assistant.vue]: Ahh good to know. Thx

**Comment by Noles** [comment]
Very nice! Thanks

**Pull request closed without merging**